.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -2,4 +2,4 @@ blank_issues_enabled: false
 contact_links:
   - name: Questions
     url: https://discord.gg/Byjr6rdBUZ
-    about: Any questions or comments you have, let us know in our Discord server.
+    about: Any questions or comments you have, let us know on our Discord server.
```

Comment:
I don't believe this change is correct.

You place a file *on* a computer server, but you interact with people *in* a discord server.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
apologize for that, I mention the error in discord.

Reply:
Good point @naomi-lgbt 

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient